api/samples/CMakeLists.txt

```diff
@@ -218,9 +218,9 @@ if (X86) # FIXME i#1551, i#1569: port to ARM and AArch64
   add_sample_client(countcalls  "countcalls.c"    "drmgr;drreg")
 endif (X86)
 add_sample_client(div         "div.c"           "drmgr")
+add_sample_client(memval_simple       "memval_simple.c;utils.c" "drmgr;drreg;drutil;drx")
```

Comment:
Done (I think). Moved around some entries, but let me know if the change was unwarranted.